### PR TITLE
K8S-116 Remove magnum deployment

### PR DESCRIPTION
**Pull request by odyssey4me**
The approach for implementing containers is
going to be completely re-done. This patch
removes the current approach to prevent bit
rot.

The remaining Octavia bits are adjusted to
ensure that the SHA's are fixed at a point
in time. As the development work for that
is progressed, those SHA's may be adjusted.

(cherry picked from commit 6a4e1381858fcf5669b843e12e356f56349d891e)

Issue: [K8S-116](https://rpc-openstack.atlassian.net/browse/K8S-116)

### CODE REVIEW

**Comment by odyssey4me** [comment]
recheck_cit_all all instances failed to build

**Comment by odyssey4me** [comment]
recheck_cit_artifact
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by mattt416** [comment]
recheck_cit_artifact
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by mattt416** [comment]
The `recheck_cit_artifact` build failed with this:

```
TASK [bootstrap-host : Deploy user conf.d configuration] ***********************
task path: /opt/rpc-openstack/openstack-ansible/tests/roles/bootstrap-host/tasks/prepare_aio_config.yml:28
[DEPRECATION WARNING]: Skipping task due to undefined Error, in the future this
 will be a fatal error.: {{ confd_overrides[scenario] }}: 'dict object' has no 
attribute u'aio'.
This feature will be removed in a future release. Deprecation
 warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
fatal: [localhost]: FAILED! => {"failed": true, "msg": "the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'item' is undefined\n\nThe error appears to have been in '/opt/rpc-openstack/openstack-ansible/tests/roles/bootstrap-host/tasks/prepare_aio_config.yml': line 28, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Deploy user conf.d configuration\n  ^ here\n"}
```

**Comment by odyssey4me** [comment]
I expect that this needs a rebase. Please do not merge until 14.2.0 is released.

File: scripts/bootstrap-aio.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@
   include: "{{ lookup('env', 'OA_DIR') }}/tests/bootstrap-aio.yml"
   vars:
     bootstrap_host_apt_distribution_suffix_list: "{{ (lookup('env', 'RPCO_APT_ARTIFACTS_AVAILABLE') | bool) | ternary([], ['updates', 'backports']) }}"
-    scenario: "{% if lookup('env', 'DEPLOY_MAGNUM') == 'yes' %}magnum{% else %}{{ lookup('env', 'DEPLOY_CEPH') |bool |ternary('ceph','swift') }}{%endif%}"
+    bootstrap_host_scenario: "{{ (lookup('env', 'DEPLOY_CEPH') | bool) | ternary('ceph','swift') }}"
```
Comment (mattt416) [inline scripts/bootstrap-aio.yml]: Should this be `scenario` or `bootstrap_host_scenario`?

Comment (odyssey4me) [reply scripts/bootstrap-aio.yml]: Ah, you're right - I got it mixed up. In Newton it's scenario, in Ocata onwards it's bootstrap_host_scenario. Thanks!

**Comment by odyssey4me** [comment]
recheck_cit_artifact

**Comment by odyssey4me** [comment]
trusty_leapfrogupgrade_swift - ``Quota exceeded for ram: Requested 15360, but already used 510976 of 512000 ram``

**Comment by major** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by odyssey4me** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by odyssey4me** [comment]
recheck_cit_trusty_deploy_swift

**Comment by odyssey4me** [comment]
recheck_cit_trusty_deploy_swift - tempest test failure

**Comment by odyssey4me** [comment]
recheck_cit_trusty_deploy_swift - tempest test failure, looks like the same one

**Comment by cloudnull** [comment]
recheck_cit_trusty_deploy_swift  - one more time for good measure

**Comment by major** [comment]
recheck_cit_trusty_deploy_swift

**Pull request merged**